flatpak_builder_lint/checks/finish_args.py

```diff
@@ -69,6 +69,24 @@ def _validate(self, appid: Optional[str], finish_args: dict[str, Set[str]]) -> N
                     own_name.startswith(appid) and own_name[len(appid)] == "."
                 ):
                     self.errors.add("finish-args-unnecessary-appid-own-name")
+            if own_name == "org.freedesktop.*":
+                self.errors.add("finish-args-wildcard-freedesktop-own-name")
+            if own_name == "org.gnome.*":
+                self.errors.add("finish-args-wildcard-gnome-own-name")
+            if own_name == "org.kde.*":
+                self.errors.add("finish-args-wildcard-kde-own-name")
+            if own_name.startswith("org.freedesktop.portal"):
+                self.errors.add("finish-args-portal-own-name")
+
+        for talk_name in finish_args["talk-name"]:
+            if talk_name == "org.freedesktop.*":
+                self.errors.add("finish-args-wildcard-freedesktop-talk-name")
+            if talk_name == "org.gnome.*":
+                self.errors.add("finish-args-wildcard-gnome-talk-name")
+            if talk_name == "org.kde.*":
+                self.errors.add("finish-args-wildcard-kde-talk-name")
+            if talk_name.startswith("org.freedesktop.portal"):
```

Comment:
```suggestion
            if talk_name.startswith("org.freedesktop.portal."):
```

I'd add the period here. and also above.